typescript/example_code/README.rst

```diff
@@ -0,0 +1,14 @@
+###############################
+TypeScript AWS SDK examples
+###############################
+This repo contains TypeScript code examples for the following AWS Services:
```

Comment:
"the following AWS Services:" -> "the following:" -- i.e. the CDK is not, strictly speaking a "service".